===================================================================
RCS file: /cvs/dbus/dbus/mono/BusDriver.cs,v
retrieving revision 1.1
retrieving revision 1.2
diff -u -r1.1 -r1.2
--- dbus/dbus/mono/BusDriver.cs	2004/08/31 03:59:14	1.1
+++ dbus/dbus/mono/BusDriver.cs	2004/09/29 01:46:45	1.2
@@ -3,7 +3,9 @@
 
   using System;
 
-  public delegate void ServiceEventHandler (string serviceName);
+  public delegate void ServiceEventHandler (string serviceName,
+					    string oldOwner,
+					    string newOwner);
 
   [Interface ("org.freedesktop.DBus")]
   public abstract class BusDriver
@@ -19,11 +21,7 @@
 
 
     [Signal]
-    public virtual event ServiceEventHandler ServiceCreated;
-
-    [Signal]
-    public virtual event ServiceEventHandler ServiceDeleted;
-
+    public virtual event ServiceEventHandler ServiceOwnerChanged;
 
     static public BusDriver New (Connection connection)
     {
